### DISCUSSION TITLE
My Github account is not publicly visible anymore

### DISCUSSION THREAD

**Discussion by haroon-wyxan** [community/community#121612] [New to GitHub]
My profile [haroonrashid2210](https://github.com/haroonrashid2210)

I got an email notification three days ago. As a result, my profile is no longer visible to others, they cannot see issues I've created, and I cannot be assigned to pull requests.

<img width="993" alt="image" src="https://github.com/community/community/assets/45206179/86f80a6d-7b3d-4410-b6c1-3d402f474269">

<img width="913" alt="image" src="https://github.com/community/community/assets/45206179/e91bc288-9ca7-4101-a37f-125738f1a75d">

After that, I get logged out from everywhere (github.com, GitHub desktop, vs code, etc)

I then checked my security logs and found out

![image](https://github.com/community/community/assets/45206179/5ae4d0d2-d4aa-4573-98c1-fb847c5d364a)

I created a support ticket but seeing the community discussions I don't think Github is actively resolving these issues.

<img width="1031" alt="image" src="https://github.com/community/community/assets/168262832/743c8e6c-6848-47ce-8cf0-4a5a3c92b90e">

- https://github.com/orgs/community/discussions/55609
- https://github.com/orgs/community/discussions/27294
- https://github.com/orgs/community/discussions/58867
- https://github.com/orgs/community/discussions/27294
- https://github.com/orgs/community/discussions/31353
- https://github.com/orgs/community/discussions/23842

And there are more related issues. 

I would appreciate it if the GitHub support team could assist me as soon as possible. It is significantly impacting my professional life. 😢

**Comment by Bhoovi333** [comment]
hi, go on repository option, you will simply see private and public option. do it public.

**Comment by yogeshxd** [comment]
I got the same issue and it's still not resolved. Github is not even responding to these issues.

**Comment by hassanalimalek** [comment]
This is a frustrating experience on GitHub.  Being unable to see assigned issues or participate in pull requests due to a hidden profile significantly impacts productivity. Github should fix this issue.

**Comment by ebndev** [comment]
Hi @haroon-wyxan,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion and any duplicate discussions.

We appreciate your understanding!
